desktop_version/src/BinaryBlob.cpp

```diff
@@ -107,7 +107,7 @@ bool binaryBlob::unPackBinary(const char* name)
 	}
 	PHYSFS_close(handle);
 
-	printf("The complete reloaded file size: %li\n", size);
+	printf("The complete reloaded file size: %li\n", (long int)size);
```

Comment:
This might warn on type size issues... should we change the printf type instead?

Reply:
Right, I'll change it to `%lli`.

Reply:
Good point, I changed it to `%lli`.